### ISSUE TITLE
Is the project still active?

### ISSUE THREAD

**Issue by alejandroiglesias** [arranger1044/iSoul#42]
I know this isn't strictly an issue, but is the project still active? It would be good if it is, I would love to share some UI enhancements based on my own experience and that of the users on the iSoul channel, though I cannot code in C/Obj-C.
Please let me know.
Cheers.

**Comment by arranger1044** [comment]
Yep the project is still alive, even though it is sleeping at the moment : )
Be glad to share your ideas, since this is an untold rule from the free software principles we adhere to. We cannot guarantee we would be the ones that put it right in code form anyway.

**Comment by interconnect** [comment]
Yeah it'd be great to see some activity resume on this project!

A new build on the Downloads page with the latest fixes would be awesome too!

**Comment by arranger1044** [comment]
I am a bit overburdened by life issues at the moment thus I do not have so much spare time.
@djbe where you having some issue with the sparkle release?

**Comment by Smultie** [comment]
Might be a good time to do a small bump, maybe?

**Comment by ghost** [comment]
Yes. 8)

**Comment by arranger1044** [comment]
Hi everyone, I've had a hard time with my studies, plus I have to get my old mac back (I guess I can install mavericks on it). I hope to find some time in the next holidays to recompile it for mavericks at least.
Btw, what you could do by now is trying to spread the word among OS X developers.

**Comment by ghost** [comment]
Hi ! Good luck with your studies.
iSoul nightly build 16-05-12 works great on Mavericks. I'm using it on Mavericks since last year, as far I know, no bugs.
My only whises are an ignore feature and maybe a new layout for the rooms, smaller lines and more colors (sometimes, 2/3 users have the same color, it's confusing). Oh, and an IP identification feature too, rooms are filed with trolls, it's awful.

**Comment by arranger1044** [comment]
Thanks, btw every contribution will be appreciated, even a mock up of an interface redesign or some new icons. Try to spread the word not to have this project dying

**Comment by ghost** [comment]
I'll try some graphic stuff. =)

**Comment by Smultie** [comment]
Small kick ;) 

Sorry !

**Comment by arranger1044** [comment]
I am alive but the project is still on hold, I am now programming only on a linux machine, still waiting for someone willing to lend a hand : )

**Comment by Smultie** [comment]
Good to know you're still alive, that's the most important part of living, right ;)

What exactly do you need help on?

**Comment by arranger1044** [comment]
well, to be quite honest the whole project could be rewritten according to the latest libs and frameworks, however it would suffice to mantain it and render it stable on the latest version of OS X (and fix some annoying bugs)

**Comment by Smultie** [comment]
Sooo @arranger1044 , any plans to do this "rewritten according to the latest libs and frameworks, however it would suffice to mantain it and render it stable on the latest version of OS X (and fix some annoying bugs)" stuff?

**Comment by arranger1044** [comment]
I barely have the time to do the side courses at my university at the moment. 
I still have hopes that someone will show to get this project going.

**Comment by mo-seph** [comment]
I had a go a building the current version, but there are lots of deprecated functions in (e.g.) the libevent library. I'm not much of an xcode user, so I'm a bit at a loss how to replace libraries with newer versions.

**Comment by Smultie** [comment]
Hey @mo-seph 
Really appreciate someone trying to update iSoul!! Unfortunately I'm no programmer, so I can't help you there, but maybe my post will get some attention from people that can!
